src/index.js

```diff
@@ -1,42 +1,85 @@
 import './styles.css';
 
-const refreshButton = document.getElementById('refreshButton');
-const addButton = document.getElementById('addButton');
-const scoreboard = document.getElementById('scoreboard');
-const nameInput = document.getElementById('nameInput');
-const scoreInput = document.getElementById('scoreInput');
-
-// Load scores from local storage or initialize if not found
-const scores = JSON.parse(localStorage.getItem('scores')) || [
-  { name: 'Alice', score: 100 },
-  { name: 'Bob', score: 85 },
-  { name: 'Carol', score: 120 },
-];
-
-function refreshScoreboard() {
-  scoreboard.innerHTML = '';
-  scores.forEach((entry, index) => {
-    const entryDiv = document.createElement('div');
-    entryDiv.innerHTML = `${index + 1}. ${entry.name}: ${entry.score}`;
-    scoreboard.appendChild(entryDiv);
-  });
-}
-
-function addScore() {
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+let gameID = '';
+
+const createGame = async () => {
+  try {
+    const response = await fetch(`${baseURL}/games/`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: 'BasketBall',
+      }),
+    });
+    const data = await response.json();
+    const [, gameIDPart] = data.result.split(': ');
+    gameID = gameIDPart;
+    console.log('Game created:', gameID);
+  } catch (error) {
+    console.error('Error creating game:', error);
+  }
+};
+
+const refreshScoreboard = async () => {
+  try {
+    const response = await fetch(`${baseURL}/games/${gameID}/scores/`);
+    const data = await response.json();
+    const scoreboard = document.getElementById('scoreboard');
+
+    const maxLength = data.result.reduce(
+      (max, entry) => Math.max(max, entry.user.length),
+      0,
+    );
+
+    scoreboard.innerHTML = data.result
+      .map((entry, index) => {
+        const evenClass = index % 2 !== 0 ? 'even' : '';
+        return `<div class="entry ${evenClass}">
+                  <span class="name">${entry.user.padEnd(maxLength, ' ')}</span>
+                  <span class="separator">:</span>
+                  <span class="score">${entry.score}</span>
+                </div>`;
+      })
+      .join('');
+  } catch (error) {
+    console.error('Error refreshing scoreboard:', error);
+  }
+};
```

Comment:
- [ ] Dear @Rafiduzzaman, you did a really great job by implementing the features given in the project requirements and you beautifully used JavaScript in order to achieve better results. But while testing your application on my side by running a live server, I figured out a few problems with your approach, which are listed below:

1. Your `create game` function because you are calling the function at line number **85**, every time one application loads for the first time or even on the refresh button your `gameID` actually change every time which actually causes some issues like:
     - You are misusing the API resource by hitting the API on every refresh or change in the data.
     - Because you are generating the new ID every time, the data will not populate in your table because once you added the data it is saved to the generated ID and by the next time your ID change.

2. Make sure that you have only one generated ID and you are using that same *ID* for posting the data and getting the data from the API.

After implementing the above mentioned changes, I would like to suggest your to cross check your application before next submission and ensure that your data is visible on the table whenever you are submitting new data from the form.


### HINTS

**You can use the in-built javascript function to call your function while onload event, and when any person click on refresh button then use window load function, which will automatically invoke your function to sync the data into table from API.**